### PR TITLE
Add Handling of Zero Len Arguments

### PR DESCRIPTION
**Pull request by codeislife99**
Currently there is a bug in the add operator for TRT where we receive a zero-length add operator. This PR is to offload add and even any such future ops which might have zero length to VM.

### CODE REVIEW

**Comment by codeislife99** [comment]
@anijain2305 @trevor-m @zhiics PTAL

**Comment by trevor-m** [comment]
LGTM, I would clarify that the bug is in the TVM frontend somewhere which is creating an add op with no args.

**Review by anijain2305 — APPROVED** [review]

File: python/tvm/relay/tensorrt.py [hunk 1]
```diff
@@ -119,6 +119,10 @@ def check_dynamism(args, op_name):
     ----------
     True if dynamism is present, False otherwise
     """
+    if len(args) == 0:
```
Comment (zhiics) [inline python/tvm/relay/tensorrt.py]: nitpick: python prefers `if not args:`

Comment (codeislife99) [reply python/tvm/relay/tensorrt.py]: Done.

**Review by zhiics — APPROVED** [review]
please fix the ci error.

**Pull request merged**